android/src/main/kotlin/com/onegini/mobile/sdk/flutter/useCases/DeregisterAuthenticatorUseCase.kt

```diff
@@ -49,4 +49,4 @@ class DeregisterAuthenticatorUseCase(private var oneginiClient: OneginiClient) {
         return authenticator
     }
 
-}
\ No newline at end of file
+}
```

Comment:
no empty lines at the end of the few files below 👇

Reply:
done